Fakes/Fakes/Networking.generated.swift

```diff
@@ -1153,6 +1153,8 @@ extension Networking.POSProduct {
             regularPrice: .fake(),
             salePrice: .fake(),
             onSale: .fake(),
+            downloadable: .fake(),
```

Comment:
`downloadable` is filtered for most POS requests, but not for scanning. It will be needed to check whether we should add things to the cart, because if not, we will show an error which says _why_ it couldn't be.